components/Card.js

```diff
@@ -3,17 +3,15 @@ import styled from "styled-components";
 import { EntertainmentIcon } from "@/components/Icons";
 import { AppliancesIcon } from "@/components/Icons";
 import { WorkIcon } from "@/components/Icons";
+import { LightingIcon } from "@/components/Icons";
```

Comment:
All of these imports could be merged into one line:

```suggestion
import { EntertainmentIcon, AppliancesIcon, WorkIcon, LightingIcon } from "@/components/Icons";
```